test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -410,8 +410,13 @@ def test_faster_rcnn(self):
         self.run_test(model, (x,), rtol=1e-3, atol=1e-5)
         self.run_test(model, (x,), input_names=["images_tensors"], output_names=["outputs"],
                       dynamic_axes={"images_tensors": [0, 1, 2, 3], "outputs": [0, 1, 2, 3]}, rtol=1e-3, atol=1e-5)
+        dummy_image = [torch.ones(3, 100, 100) * 0.3]
+        images, test_images = self.get_test_images()
+        self.run_test(model, (images,), test_with_inputs=[(images,), (test_images,), (dummy_image,)],
+                      input_names=["images_tensors"], output_names=["outputs"],
+                      dynamic_axes={"images_tensors": [0, 1, 2], "outputs": [0, 1, 2]}, rtol=1e-3, atol=1e-5)
 
-    def get_image_from_url(self, url):
+    def get_image_from_url(self, url, size=None):
```

Comment:
nit: size=(300, 200) and remove below?

Reply:
This piece of code is copied from torch vision test, want it to be identical...